### PR TITLE
Update contribution page with coding guide

### PR DESCRIPTION
**Pull request by akchinSTC**
Signed-off-by: Alan Chin <akchin@us.ibm.com>

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The following items will be discussed in this document :
 
 * [Contributing Documentation Changes](#Contributing+Documentation+Changes)
```
Comment (ptitzler) [inline CONTRIBUTING.md:31]: I don't think the in-doc HREF's work at all. We typically lower case the characters and replace white spaces with dashes, e.g.  `#contributing-...`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,6 +57,11 @@ or other means you agree to license the material under the project's open source
 have the legal authority to do so.**
 ***
 
+## Coding style guide
+
+Elyra has auto-linting in place in its CI process. We use flake8 for all python (PEP-8 with google style imports) and
```
Comment (ptitzler) [inline CONTRIBUTING.md:62]: nit: markdown

```suggestion
Elyra has auto-linting in place in its CI process. We use `flake8` for all python (PEP-8 with google style imports) and 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -56,6 +57,11 @@ or other means you agree to license the material under the project's open source
 have the legal authority to do so.**
 ***
 
+## Coding style guide
+
+Elyra has auto-linting in place in its CI process. We use flake8 for all python (PEP-8 with google style imports) and 
+a combination of eslint (rules can be found in repo respective .eslintrc.json file) and prettier for our Javascript/Typescript.
```
Comment (ptitzler) [inline CONTRIBUTING.md:63]: nit: markdown

```suggestion
a combination of `eslint` (rules can be found in repo respective `.eslintrc.json` file) and `prettier` for our Javascript/Typescript.
```

**Review by ptitzler — CHANGES_REQUESTED** [review]
See HREFs

**Comment by akchinSTC** [comment]
@ptitzler - made requested changes

**Review by ptitzler — APPROVED** [review]
LGTM!

As discussed offline, we might want to also document that sign-offs are now required for each commit, or DCO will fail.

**Pull request merged**